banzai/logs.py

```diff
@@ -1,21 +1,45 @@
-from __future__ import absolute_import, division, print_function, unicode_literals
-
-import logutils.queue
 import logging
 import multiprocessing
 import sys
+import os
+
+import logutils.queue
 from lcogt_logging import LCOGTFormatter
+
 from banzai.utils import date_utils
 
 queue = None
 listener = None
 
 
-def get_logger(name):
-    logger = logging.getLogger(name)
-    # This allows us to control the logging level with the root logger.
-    logger.setLevel(logging.DEBUG)
-    return logger
+class BanzaiLogger(logging.getLoggerClass()):
+    def __init__(self, name, level='NOTSET'):
+        super(BanzaiLogger, self).__init__(name, level)
+
+    def _log(self, level, msg, *args, **kwargs):
+        kwargs = _add_tag_dictionary(kwargs)
+        super(BanzaiLogger, self)._log(level, msg, *args, **kwargs)
+
+
+def _add_tag_dictionary(kwargs):
```

Comment:
maybe _add_tag_to_dictionary? You might also want to say this is logging tags to dictionary.

Reply:
I agree that the name was not clear, but I have now called it `create_logging_tags_dictionary` because I think that describes what it's really doing